lib/Service/Appointments/MailService.php

```diff
@@ -217,21 +217,24 @@ private function addBulletList(IEMailTemplate $template,
 		bool $recipient): void {
 		$template->addBodyListItem($booking->getDisplayName(), $l10n->t('Appointment for:'));
 
-		// determain timezone depending on who is getting the message (Requestee/Requester)
+		// determine timezone depending on who is getting the message (Requestee/Requester)
 		$tzid = ($recipient) ? $config->getAvailabilityAsArray()['timezoneId'] : $booking->getTimezone();
 		$dtstart = new \DateTime('now', new \DateTimeZone($booking->getTimezone())); // generate DateTime with booking time zone
 		$dtstart->setTimestamp($booking->getStart()); // set booking time stamp
 
 		$l = $this->lFactory->findGenericLanguage();
+		$relativeTimezone = new \DateTimeZone($tzid);
 		$relativeDateTime = $this->dateFormatter->formatDateTimeRelativeDay(
 			$dtstart,
 			'long',
 			'short',
-			new \DateTimeZone($tzid),
+			$relativeTimezone,
 			$this->lFactory->get('calendar', $l)
 		);
 
-		$template->addBodyListItem($relativeDateTime, $l10n->t('Date:'));
+		$timestring = $relativeDateTime . ' (' . $relativeTimezone->getName() . ')';
```

Comment:
A nice future improvement we could have is using `IntlTimeZone::getDisplayName` to have the timezone name localized, if the `intl` module is available.